src/imex.rs

```diff
@@ -462,7 +462,9 @@ fn import_backup(context: &Context, backup_to_import: impl AsRef<Path>) -> Resul
         |files| {
             for (processed_files_cnt, file) in files.enumerate() {
                 let (file_name, file_blob) = file?;
-                ensure!(!context.shall_stop_ongoing(), "received stop signal");
+                if context.shall_stop_ongoing() {
+                    return Ok(false);
+                }
```

Comment:
isn't that a functional change? Previously an error was returned and now Ok(false)?

Reply:
I thought that too the first time.  But this is matched on below with `Ok(all_files_extracted)` and the same error return is used there for this case.